spec/Variables.tex

```diff
@@ -21,8 +21,12 @@ \section{Variable Declarations}
 config-or-extern: one of
   `config' $ $ $ $ `extern'
 
-variable-kind: one of
-  `param' $ $ $ $ `const' $ $ $ $ `var'
+variable-kind:
+  `param'
+  `const'
+  `var'
+  `ref'
+  `const' `ref'
```

Comment:
I would have thought you would write this as
      `const ref'
and that would be better for grepping. I'd imagine it does not look different in the output.